src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search__container">
+      <form
+        action="/"
+        method="get"
+        class="search-bar"
+        data-qa="big"
+      >
+        <div class="icon">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'icon' could be more descriptive.